azure-pipelines.yml

```diff
@@ -24,13 +29,15 @@ jobs:
     dependsOn:
       - PrepareNode
     displayName: Maven Build (Container)
-    pool: cauldron-vmss
+    pool: cybertron-build-pool
     variables:
       containerRegistry: uchimera
       imageRepository: cccs/trino-base
       buildTimestamp: $[format('{0:yyyyMMddHHmmss}', pipeline.startTime)]
-      JAVA_TOOL_OPTIONS: " -Xmx8g"
       DOCKER_BUILDKIT: 1
+      MAVEN_INSTALL_OPTS: "-Xmx4G -XX:+ExitOnOutOfMemoryError"
+      MAVEN_FAST_INSTALL: "-B -V -T 1C -DskipTests -Dmaven.source.skip=true"
+      SKIPPED_PROJECTS: "!:trino-docs,!:trino-server-rpm,!:trino-blackhole,!:trino-exasol,!:trino-faker,!:trino-hdfs,!:trino-hive,!:trino-hudi,!:trino-ignite,!:trino-kafka,!:trino-kafka-event-listener,!:trino-kinesis,!:trino-ml,!:trino-phoenix5,!:trino-pinot,!:trino-snowflake,!:trino-example-http,!:trino-google-sheets,!:trino-druid,!:trino-exchange-hdfs,!:trino-openlineage,!:trino-jmx,!:trino-thrift-api,!:trino-thrift-testing-server,!:trino-thrift,!:trino-clickhouse,!:trino-spooling-filesystem,!:trino-opensearch,!:trino-vertica,!:trino-singlestore,!:trino-teradata-functions,!:trino-elasticsearch,!:trino-kudu,!:trino-bigquery,!:trino-testing-resources,!:trino-delta-lake,!:trino-prometheus,!:trino-cassandra,!:trino-sqlserver,!:trino-mariadb,!:trino-redis,!:trino-redshift,!:trino-example-jdbc,!:trino-proxy,!:trino-verifier,!:trino-benchmark-queries,!:trino-benchto-benchmarks,!:trino-tests,!:trino-faulttolerant-tests,!:trino-plugin-reader,!:trino-product-tests-groups,!:trino-product-tests,!:trino-product-tests-launcher,!:trino-server-dev,!:trino-test-jdbc-compatibility-old-driver,!:trino-test-jdbc-compatibility-old-server,!:trino-testing-services,!:trino-testing,!:trino-testing-containers,!:trino-testing-kafka"
```

Comment:
We use `trino-jmx`, don't we? I think I would also leave any plugins for things that are already in our stack, just in case. Like `trino-kafka-*`, `trino-elasticsearch` and maybe even `trino-prometheus`.

Reply:
trino-jmx shouldn't be on that list, I'll remove it. The other 3 I deliberately put them there because we gave up on that one elasticsearch catalog we had and we've never had kafka and prometheus. The 3 projects you mentioned took 30 seconds, 17 seconds and 6 seconds to build respectively (+ some additional time for provisioning).